gst/isomp4/qtdemux.c

```diff
@@ -5209,16 +5118,12 @@ qtdemux_parse_moov (GstQTDemux * qtdemux, const guint8 * buffer, guint length)
   qtdemux_parse_node (qtdemux, qtdemux->moov_node, buffer, length);
 
   pssh = qtdemux_tree_get_child_by_type (qtdemux->moov_node, FOURCC_pssh);
+#ifdef HAVE_FLUC
   if (pssh) {
-    GstBuffer *buf;
-
-    /* for the pssh, just send the buffer completely */
-    buf = gst_buffer_new ();
-    GST_BUFFER_DATA (buf) = pssh->data;
-    GST_BUFFER_SIZE (buf) = GST_READ_UINT32_BE (pssh->data);
-    g_signal_emit (qtdemux, gst_qtdemux_signals[SIGNAL_PSSH], 0, buf);
-    gst_buffer_unref (buf);
+    GstEvent *event = fluc_drm_event_new_pssh (pssh->data, "isobmff/moov");
```

Comment:
fluc_drm_event_new_pssh can return NULL

Reply:
I will print an error if the event could not be created

Reply:
Done